product_docs/docs/epas/11/uninstalling/linux_uninstall.mdx

```diff
@@ -1,5 +1,8 @@
 ---
-title: "Uninstalling Advanced Server"
+title: "Uninstalling Advanced Server on Linux"
+navTitle: "On Linux"
+redirects:
+ - /epas/11/epas_inst_linux/06_uninstalling_advanced_server/
```

Comment:
Can we make all the titles consistent on this page like
- Uninstalling on RHEL/OL/AlmaLinux/Rocky Linux/ ..
-  Uninstalling on Debian/Ubuntu

And have a separate Updating topic?